websites/S/Simplyaweeb/iframe.ts

```diff
@@ -0,0 +1,13 @@
+const iframe = new iFrame();
+
+iframe.on("UpdateData", async () => {
+	const video: HTMLVideoElement = document.querySelector("video");
+
+	if (video && !isNaN(video.duration)) {
```

Comment:
```suggestion
	if (!isNaN(video?.duration)) {
```

Reply:
Wow thanks ! I didn't think of that way ! This will be usefull for my next presences and projects !